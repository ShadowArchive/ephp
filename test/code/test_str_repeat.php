<?php

var_dump(str_repeat([], ""));
var_dump(str_repeat("/base", ""));
var_dump(str_repeat("/base", 2));
var_dump(str_repeat("/base", 0));
var_dump(str_repeat("/base", 10));
var_dump(str_repeat("/base", -1));

