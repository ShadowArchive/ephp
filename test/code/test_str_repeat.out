
Warning: str_repeat() expects parameter 1 to be string, array given in {{CWD}}/test/code/test_str_repeat.php on line 3
NULL

Warning: str_repeat() expects parameter 2 to be long, string given in {{CWD}}/test/code/test_str_repeat.php on line 4
NULL
string(10) "/base/base"
string(0) ""
string(50) "/base/base/base/base/base/base/base/base/base/base"

Warning: str_repeat(): Second argument has to be greater than or equal to 0 in {{CWD}}/test/code/test_str_repeat.php on line 8
NULL
